<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-06-14T17:11:44 IST" name="Default suite" finished-at="2022-06-14T17:11:44 IST" duration-ms="88">
    <groups>
    </groups>
    <test started-at="2022-06-14T17:11:44 IST" name="Default test" finished-at="2022-06-14T17:11:44 IST" duration-ms="88">
      <class name="test.HomeTest">
        <test-method is-config="true" signature="initBrowser()[pri:0, instance:test.HomeTest@dd8ba08]" started-at="2022-06-14T17:11:25 IST" name="initBrowser" finished-at="2022-06-14T17:11:44 IST" duration-ms="18115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initBrowser -->
        <test-method is-config="true" signature="initObject()[pri:0, instance:test.HomeTest@dd8ba08]" started-at="2022-06-14T17:11:44 IST" name="initObject" finished-at="2022-06-14T17:11:44 IST" duration-ms="24" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initObject -->
        <test-method signature="logout()[pri:0, instance:test.HomeTest@dd8ba08]" started-at="2022-06-14T17:11:44 IST" name="logout" finished-at="2022-06-14T17:11:44 IST" duration-ms="0" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
The method assertFalse(boolean) in the type Assert is not applicable for the arguments (void)
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
The method assertFalse(boolean) in the type Assert is not applicable for the arguments (void)

at test.HomeTest.logout(HomeTest.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- test.HomeTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
